app/bundles/EmailBundle/Controller/EmailController.php

```diff
@@ -1304,7 +1304,7 @@ function (&$field) {
         ];
 
         // Send to current user
-        $errors = $model->sendEmailToUser($entity, $users, $fields, [], [], false);
+        $errors = $model->sendSampleEmailToUser($entity, $users, $fields, [], [], false);
```

Comment:
Hi 
Thanks for review.
It is lead fields so the name as fields. And it is ample mail function so get all lead fields and add key id with value 0.